packages/fluent-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -59,49 +53,35 @@ const SelectWidget = ({
 }: WidgetProps) => {
   const { enumOptions, enumDisabled } = options;
 
-  const emptyValue = multiple ? [] : '';
+  const _onChange = ({}, value: any) => onChange(value.key);
 
-  const _onChange = ({
-    target: { value },
-  }: React.ChangeEvent<{ name?: string; value: unknown }>) =>
-    onChange(processValue(schema, value));
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, processValue(schema, value));
```

Comment:
Please don't call processValue here, also please remove the actual definition of processValue

Reply:
Done.